OWNERS

```diff
@@ -1,9 +1,5 @@
 reviewers:
-- gabemontero
-- adambkaplan
-- bparees
+- coreydaley
 approvers:
-- bparees
-- gabemontero
-- adambkaplan
+- coreydaley
```

Comment:
is there anyone else from your team or perhaps QE that could be enlisted as an owner? in general we really don't want single-approver repos since it's your on PTO and a change is needed here, not only can no one approve it, but it may not be obvious who else to talk to about it.